mail-mta/opensmtpd/opensmtpd-7.4.0_p1.ebuild

```diff
@@ -57,6 +57,8 @@ QA_CONFIG_IMPL_DECL_SKIP=( closefrom )
 
 DOCS=( {CHANGES,README}.md )
 
+PATCHES=( "${FILESDIR}/${P}-missing-includes.patch" )
```

Comment:
This doesn't match the filename of your patch. Have updated.

Reply:
Thanks, that's painful oopsie.